codegen/core/src/main/java/software/amazon/smithy/python/codegen/generators/SchemaGenerator.java

```diff
@@ -83,8 +83,7 @@ public static void generateAll(
         var index = TopologicalIndex.of(context.model());
         Stream.concat(index.getOrderedShapes().stream(), index.getRecursiveShapes().stream())
                 .filter(shapes::contains)
-                .filter(shape -> !shape.isOperationShape() && !shape.isResourceShape()
-                        && !shape.isServiceShape()
+                .filter(shape -> !shape.isResourceShape()
```

Comment:
The filter in the types generator should be updated to filter out operations and services